app/models.py

```diff
@@ -750,6 +759,112 @@ def serialize(self, include_user=False):
 
         return data
 
+
+class Brief(db.Model):
+    __tablename__ = 'briefs'
+
+    STATUSES = ('draft', 'live')
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    framework_id = db.Column(db.Integer, db.ForeignKey('frameworks.id'), nullable=False)
+    _lot_id = db.Column("lot_id", db.Integer, db.ForeignKey('lots.id'), nullable=False)
+
+    data = db.Column(JSON)
+    status = db.Column(db.String, index=False, unique=False,
+                       nullable=False, default='draft')
+
+    created_at = db.Column(db.DateTime, index=True, nullable=False,
+                           default=datetime.utcnow)
+    updated_at = db.Column(db.DateTime, index=True, nullable=False,
```

Comment:
This is a bit speculative, but it feels like the most important date for briefs is the publishing date – it makes the most sense for ordering and might be useful as a base point for expiration etc. With edit endpoints we have right now, publishing date should be the same as the last updated_at. But if we plan to store clarification questions as part of the brief that will allow some updates for live briefs, so will automatically change the updated_at timestamp.

We could also use audit event to find the publishing date, but it'd be hard to use for ordering.